pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -215,14 +296,35 @@ type ResourceSummary struct {
 	// Total amount of allocatable resources on all nodes.
 	// +optional
 	Allocatable corev1.ResourceList `json:"allocatable,omitempty"`
+
 	// Allocating represents the resources of a cluster that are pending for scheduling.
 	// Total amount of required resources of all Pods that are waiting for scheduling.
 	// +optional
 	Allocating corev1.ResourceList `json:"allocating,omitempty"`
+
 	// Allocated represents the resources of a cluster that have been scheduled.
 	// Total amount of required resources of all Pods that have been scheduled to nodes.
 	// +optional
 	Allocated corev1.ResourceList `json:"allocated,omitempty"`
+
+	// AllocatableModeling represents the number of each resources modeling in a cluster that are available for scheduling.
+	// Total amount of allocatable resources on all nodes.
+	// The Grade of AllocatableModelingItem is the index of ResourceModel,
+	// The Count of AllocatableModelingItem is the number of nodes that own the resources delineated by this modeling.
+	// For example, AllocatableModelingItem{Grade: 2, Count: 10}, which means there are 10 such resource models of ResourceModel[2].
```

Comment:
Move the comments to `type AllocatableModelingItem struct`.

> which means there are 10 such resource models of ResourceModel[2]

By the way, this description is not accurate.

Reply:
Done. I have modified this description. Thanks for such a rigorous inspection.👍🏻